app/views/dashboard/index.html.erb

```diff
@@ -15,24 +15,20 @@
 
       <div class="col-lg fit-content">
         <ul class='no-bullets'>
-          <li>
-            <%= embedded_svg('icons/basic_geolocalize-01.svg', class: 'icon') %>
-            &nbsp;<%= current_user.zip_code %>
+          <li class='location'>
+            <%= current_user.zip_code %>
           </li>
```

Comment:
Now we just need to call `class='insert-bullet-type'` to get the SVG icon here.

Reply:
This is so boss!!!  Huge enhancement